database/migration/V20160712_1603__createMailingListTables.sql

```diff
@@ -26,7 +26,9 @@ INSERT INTO `mailing_lists` (`name`) VALUES ('idlers');
 INSERT INTO `mailing_lists` (`name`) VALUES ('applicants');
 INSERT INTO `mailing_lists` (`name`) VALUES ('accepted');
 INSERT INTO `mailing_lists` (`name`) VALUES ('waitlisted');
-INSERT INTO `mailing_lists` (`name`) VALUES ('attendees');
+INSERT INTO `mailing_lists` (`name`) VALUES ('software');
```

Comment:
Yeah, we originally had those three mailing lists which we opted to replace with just the attendees one.